### DISCUSSION TITLE
AZURE

### DISCUSSION THREAD

**Discussion by EnyelSalas** [Security-Onion-Solutions/securityonion#9065] [General] [answered]
Afternoon , i have to ask for some help on how to make the azure setup work , however I'M self teaching and don't have a background on IT. The problems i've found are that i can't get the install to run successfully enough to show up in the browser. i'm using the right amount of hardware and 2 NICs, but i cannot make network setup to work correctly, after so many videos on it i realized most people are doing it from vmware and digital ocean unfortunate i can download the ISO into my laptop because of low space. i'm wondering if i can just throw inside of a Virtual Machine.

**Comment by dougburks** [comment, marked as answer]
> I'M self teaching and don't have a background on IT.

Welcome! Please note that most folks learn IT before trying to learn cybersecurity.

> The problems i've found are that i can't get the install to run successfully enough to show up in the browser. i'm using the right amount of hardware and 2 NICs, but i cannot make network setup to work correctly, after so many videos on it i realized most people are doing it from vmware and digital ocean unfortunate i can download the ISO into my laptop because of low space. i'm wondering if i can just throw inside of a Virtual Machine.

Yes, you can do a simple IMPORT installation in a minimal VM with only 4GB RAM:
https://docs.securityonion.net/en/2.3/first-time-users.html

**Comment by EnyelSalas** [comment]
hey thx i really appreciate your time, i'll report back with news, i'm
really trying to unlock the next level to becoming a cyber ninja.
